app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/hacktoberfest-initiative.mdx

```diff
@@ -0,0 +1,48 @@
+---
+meta:
+  title: Hacktoberfest Initiative
+  description: An event where we encourage and support our members in contributing to and maintaining open source during Hacktoberfest.
+hero:
+  Hero: UndrawPumpkin
+tags: [memberresources]
+order: 8
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Do you know that Virtual Coffee is also open source? That's one of the reasons we participate in [DigitalOcean's Hacktoberfest](https://hacktoberfest.digitalocean.com) through our Hacktoberfest Initiative!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## What Is Hacktoberfest?
+
+Hacktoberfest is a month-long virtual event that encourages and supports open source contributions. Open Source Software (OSS) is a codebase that the public can view, contribute to, and use. Sponsored by DigitalOcean, a contributor can qualify for the official Hacktoberfest swag by registering and making four pull requests (PRs) between October 1-31.
+
+## What Is Hacktoberfest Initiative?
+
+During Hacktoberfest in October, we help and support our members to contribute to and/or maintain open sources.
+
+Our plan is to harness the power in our community to help developers become excited to contribute to Open Source Software (OSS) and some of our favorite open source repositories along the way.
+
+## How It Works
+
+Before October, we will drop the form and instructions for Hacktoberfest Initiative in some dedicated channels on Slack.
+
+Because not everyone will need the same level or type of support, we're working to accommodate as many needs as possible. This could include 1:1 mentorship, access to private Slack channels, a group coding session, a review of the project you're using for Hacktoberfest, or general community support. We're also here to cheer you on throughout the month, whether on social media, through our events, or Slack.
+
```

Comment:
Maybe link to youtube playlist from last year somewhere in here? https://www.youtube.com/watch?v=A8iI2jlsqfs&list=PLh9uT23TA65hH8sfprU1XOvT4NBesokna&ab_channel=VirtualCoffee

Reply:
Maybe we can put the link to `general community support`?

Like:
This could include 1:1 mentorship, access to private Slack channels, a group coding session, a review of the project you're using for Hacktoberfest, or [general community support](https://www.youtube.com/watch?v=A8iI2jlsqfs&list=PLh9uT23TA65hH8sfprU1XOvT4NBesokna&ab_channel=VirtualCoffee).